SumoLogic.Logging.Common/Sender/SumoLogicMessageSender.cs

```diff
@@ -182,7 +203,17 @@ public void TrySend(string body, string name)
 
             using (var httpContent = new StringContent(body, Encoding.UTF8, TextPlainMediaType))
             {
-                httpContent.Headers.Add("X-Sumo-Name", name);

```

Comment:
I would like to keep the model consistent -- instead of setting category and host via properties, but passing name as an arugment, let's just make everything a property.